worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -726,6 +726,7 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
         Math.min(diskInfo.configuredUsableSpace - totalUsage, fileSystemReportedUsableSpace)
       logDebug(s"updateDiskInfos  workingDirUsableSpace:$workingDirUsableSpace filemeta:$fileSystemReportedUsableSpace conf:${diskInfo.configuredUsableSpace} totalUsage:$totalUsage")
```

Comment:
We should also output total space in the log, although it can be calculated from `totalUsage + workingDirUsableSpace`.

Reply:
Already log all the diskInfos at the end, see 

> logInfo(s"Updated diskInfos:\n${disksSnapshot().mkString("\n")}")